packages/kbn-plugin-generator/sao_template/sao.js

```diff
@@ -47,19 +45,14 @@ module.exports = function({ name }) {
         message: 'Should an app component be generated?',
         default: true,
       },
-      generateTranslations: {
-        type: 'confirm',
-        message: 'Should translation files be generated?',
-        default: true,
-      },
-      generateHack: {
+      generateApi: {
         type: 'confirm',
-        message: 'Should a hack component be generated?',
+        message: 'Should a server API be generated?',
         default: true,
       },
-      generateApi: {
+      generateTranslations: {
         type: 'confirm',
-        message: 'Should a server API be generated?',
+        message: 'Should translation files be generated?',
         default: true,
```

Comment:
Translations are now used/handled right now, correct? should probably disable this prompt until then?